scst-store/api-walkthrough.hbs.md

```diff
@@ -12,7 +12,7 @@ The following procedure explains how to use CURL to POST an image report.
     kubectl port-forward service/metadata-store-app 8443:8443 -n metadata-store
     ```
 
-2. Retrieve the `metadata-store-read-write-client` access token. Ensure the Service Account is [created](create-service-account-access-token.md). Run:
+2. Retrieve the `metadata-store-read-write-client` access token. Ensure the Service Account is [created](create-service-account.hbs.md). Run:
 
     ```console
     export METADATA_STORE_ACCESS_TOKEN=$(kubectl get secrets metadata-store-read-write-client -n metadata-store -o jsonpath="{.data.token}" | base64 -d)
```

Comment:
I can't comment far down enough 😭 

But I noticed that the instructions here are referring to a non-ingress set-up. Shouldn't it be assuming an ingress setup?